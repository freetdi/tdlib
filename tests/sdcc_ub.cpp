
#define GDIR boost::undirectedS
#define TDIR boost::bidirectionalS
#include "sdcc_.cpp"
