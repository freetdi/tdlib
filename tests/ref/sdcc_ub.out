==test graph==
0 <--> 1 13 7 8 
1 <--> 0 14 2 9 
2 <--> 1 10 3 8 
3 <--> 2 15 4 9 
4 <--> 3 10 11 5 
5 <--> 4 12 14 6 
6 <--> 5 11 13 7 
7 <--> 0 6 12 15 
8 <--> 0 2 12 14 
9 <--> 1 3 11 13 16 
10 <--> 12 15 2 4 
11 <--> 14 4 6 9 
12 <--> 10 5 7 8 
13 <--> 0 15 6 9 
14 <--> 1 11 5 8 
15 <--> 10 13 3 7 
16 <--> 9 
17 <--> 
18 <--> 
19 <--> 
20 <--> 
====
PP+FI+TM
0 --> 17 18 19 20 
1 --> 
2 --> 
3 --> 16 
4 --> 
5 --> 
6 --> 
7 --> 1 3 4 5 
8 --> 0 2 6 7 
9 --> 8 
10 --> 9 
11 --> 10 
12 --> 11 
13 --> 12 
14 --> 13 
15 --> 14 
16 --> 
17 --> 
18 --> 
19 --> 
20 --> 
8
0: 3 7 10 13 15
1: 1 5 8 11 14
2: 5 7 8 10 12
3: 1 3 9 11 13
4: 5 6 7 11 13
5: 3 4 5 10 11
6: 1 2 3 8 10
7: 1 3 5 7 8 10 11 13
8: 1 3 5 7 8 10 13
9: 1 3 5 7 8 13
10: 1 3 7 8 13
11: 0 1 7 8 13
12: 0 1 7 8
13: 0 1 7
14: 0 1
15: 0
16: 9 16
17: 17
18: 18
19: 19
20: 20
FI
0 --> 
1 --> 
2 --> 
3 --> 
4 --> 
5 --> 4 
6 --> 
7 --> 
8 --> 
9 --> 
10 --> 
11 --> 
12 --> 7 8 9 11 
13 --> 
14 --> 10 12 13 
15 --> 6 14 
16 --> 5 15 
17 --> 16 
18 --> 17 
19 --> 18 
20 --> 0 1 2 3 19 
8
0: 17
1: 18
2: 19
3: 20
4: 9 16
5: 1 3 9 11 13
6: 5 6 7 11 13
7: 3 7 10 13 15
8: 3 4 5 10 11
9: 5 7 8 10 12
10: 0 1 7 8 13
11: 1 2 3 8 10
12: 1 3 5 7 8 10 11 13
13: 1 5 8 11 14
14: 1 3 5 7 8 11 13
15: 1 3 5 7 11 13
16: 1 3 7 11 13
17: 1 3 7 13
18: 3 7 13
19: 3 13
20: 13
thorup
0 --> 1 
1 --> 2 
2 --> 3 
3 --> 4 
4 --> 5 
5 --> 6 
6 --> 7 
7 --> 8 
8 --> 9 
9 --> 10 
10 --> 11 
11 --> 12 
12 --> 13 
13 --> 14 
14 --> 15 
15 --> 16 
16 --> 17 
17 --> 18 
18 --> 19 
19 --> 20 
20 --> 21 
21 --> 
11
0:
1: 20
2: 19
3: 18
4: 17
5: 16
6: 0 16
7: 0 15 16
8: 0 10 15 16
9: 0 10 14 15 16
10: 0 10 13 14 15 16
11: 0 10 12 13 14 15 16
12: 0 10 11 12 13 14 15 16
13: 0 9 10 11 12 13 14 15 16
14: 0 8 9 10 11 12 13 14 15
15: 0 7 8 9 10 11 12 13 14 15
16: 0 6 7 8 9 10 11 12 13 14 15
17: 0 5 6 8 9 10 11 12 14 15
18: 0 4 5 8 9 10 11 14 15
19: 0 3 4 8 9 10 14 15
20: 0 2 3 8 9 10 14
21: 0 1 2 9 14
FI
0 --> 
1 --> 
2 --> 
3 --> 
4 --> 
5 --> 4 
6 --> 
7 --> 
8 --> 
9 --> 
10 --> 
11 --> 
12 --> 7 8 9 11 
13 --> 
14 --> 10 12 13 
15 --> 6 14 
16 --> 5 15 
17 --> 16 
18 --> 17 
19 --> 18 
20 --> 0 1 2 3 19 
8
0: 17
1: 18
2: 19
3: 20
4: 9 16
5: 1 3 9 11 13
6: 5 6 7 11 13
7: 3 7 10 13 15
8: 3 4 5 10 11
9: 5 7 8 10 12
10: 0 1 7 8 13
11: 1 2 3 8 10
12: 1 3 5 7 8 10 11 13
13: 1 5 8 11 14
14: 1 3 5 7 8 11 13
15: 1 3 5 7 11 13
16: 1 3 7 11 13
17: 1 3 7 13
18: 3 7 13
19: 3 13
20: 13
PP+MD
0 --> 17 18 19 20 
1 --> 
2 --> 
3 --> 16 
4 --> 
5 --> 
6 --> 
7 --> 
8 --> 0 3 4 7 
9 --> 1 5 8 
10 --> 2 6 9 
11 --> 10 
12 --> 11 
13 --> 12 
14 --> 13 
15 --> 14 
16 --> 
17 --> 
18 --> 
19 --> 
20 --> 
8
0: 3 7 10 13 15
1: 1 5 8 11 14
2: 5 7 8 10 12
3: 1 3 9 11 13
4: 5 6 7 11 13
5: 3 4 5 10 11
6: 1 2 3 8 10
7: 0 1 7 8 13
8: 1 3 5 7 8 10 11 13
9: 1 3 5 7 8 10 11
10: 1 3 5 7 8 10
11: 1 3 5 7 8
12: 1 3 5 7
13: 1 3 5
14: 1 3
15: 1
16: 9 16
17: 17
18: 18
19: 19
20: 20
ppta
incomplete ./treedec/bits/trie.hpp:336:TRIE
0 --> 1 11 12 13 14 
1 --> 2 10 
2 --> 3 4 5 
3 --> 
4 --> 
5 --> 6 7 8 9 
6 --> 
7 --> 
8 --> 
9 --> 
10 --> 
11 --> 
12 --> 
13 --> 
14 --> 
7
0:
1: 4 5 6 9 11 12 14
2: 0 4 6 9 12 14 15
3: 0 6 9 13 15
4: 0 6 7 12 15
5: 0 2 4 9 12 14 15
6: 2 4 10 12 15
7: 0 2 8 12 14
8: 2 3 4 9 15
9: 0 1 2 9 14
10: 9 16
11: 17
12: 18
13: 19
14: 20
PP+FI
0 --> 16 18 19 20 21 
1 --> 
2 --> 
3 --> 
4 --> 
5 --> 17 
6 --> 
7 --> 
8 --> 1 3 4 7 
9 --> 
10 --> 2 5 8 9 
11 --> 6 10 
12 --> 11 
13 --> 12 
14 --> 13 
15 --> 14 
16 --> 15 
17 --> 
18 --> 
19 --> 
20 --> 
21 --> 
8
0:
1: 3 7 10 13 15
2: 5 6 7 11 13
3: 5 7 8 10 12
4: 3 4 5 10 11
5: 1 3 9 11 13
6: 0 1 7 8 13
7: 1 2 3 8 10
8: 1 3 5 7 8 10 11 13
9: 1 5 8 11 14
10: 1 3 5 7 8 11 13
11: 1 3 5 7 8 13
12: 1 3 5 7 13
13: 1 3 7 13
14: 3 7 13
15: 3 7
16: 3
17: 9 16
18: 17
19: 18
20: 19
21: 20
